src/sagemaker/workflow/pipeline.py

```diff
@@ -55,6 +55,7 @@
     "Not Executed": "grey",
     "Stopped": "purple",
     "Stopping": "purple",
+    "Starting": "green",
```

Comment:
After a second thought, it would be weird that the colors change from green for starting, blue for executing and then go back to green for succeeded

Let's use blue for starting as well